docs/beta/grain.md

```diff
@@ -1,30 +1,44 @@
+Grain is a community-specific digital currency which is issued on the basis of [Cred](/docs/beta/cred) scores. Grain represents how much a participant has supported that community, either through contributions or financial support. Grain can be used to pay participants, or for “Boosting” contributions or initiatives. 
 
 Boosting allows a participant to burn Grain in order to increase the Cred of a specific contribution. Boosting acts as a way of helping to set the project’s values and priorities, and gives the Booster an increased say in the direction of the project. When you use Grain for Boosting, you get a share of the Cred for whatever you are Boosting, which in turn will earn you future Grain.
 
-Projects can choose to issue their Grain as a cryptocurrency or token, for example by issuing Grain as an ERC-20 token on Ethereum. This allows Grain to be a conduit with which a project can pay or financially reward contributors.
+Projects can choose to issue their Grain as a cryptocurrency or token, for example by issuing Grain as an ERC-20 token on Ethereum. This allows Grain to be a conduit with which a project can pay or financially reward participants.
 
 Note that Grain does not _have_ to be tied to any actual monetary concept, such as cryptocurrency. A SourceCred project can name it “Points” and use it solely as a scoring system in addition to Cred values.
 
 ## How you earn Grain
 
-Grain is created during what we call “Harvests.” A Harvest is a periodic cycle in which the project produces new Grain tokens, and distributes those to contributors based on their Cred scores. Your Cred score determines the percentage of Grain you receive out of the total amount allocated to the community. 
+Grain is created during what we call “Harvests.” A Harvest is a two-fold process with periodic cycles, which can be configured independently:
 
-Grain Harvests can be allocated to a community as often or as seldom as that community wishes. Furthermore, each Grain distribution (Harvest) can vary in amount. How the Grain is produced and distributed is up to the project. For example, a project could produce a fixed amount of Grain every week and distribute it to contributors based on their recent contributions, or it could have a target “Grain-per-Cred” payout ratio, and Harvest enough to ensure that all contributors receive at least that amount.
+#### 1. Cred Calculation
+In this cycle, the project recomputes the Cred Contribution Graph by gathering the data your plugins are tracking, and calculating new Cred scores. By default, this cycle runs every 6 hours.
+
+#### 2. Grain Distribution
+In this less-frequent cycle, the project distributes Grain based on participants’ newest Cred scores. By default, this cycle runs once a week.
+
+Your Cred score determines the percentage of Grain you receive out of the total amount allocated to the community for each Harvest, regardless of how frequently Cred Calculation happens within it. 
```

Comment:
Suggest removing these two paragraphs. I found them quite confusing with the talk of "periodic cycles", "less-frequent cycles", etc. I think just the top level overview I suggested above would be fine, maybe with a bit more info like:

By default, projects re-compute Cred every 6 hours, but only distribute Grain once a week: on Sunday morning, UTC time. Thus, the project has a full harvest only once per week.

Reply:
Sounds good, I made the change via Review below (I can't force-commit it myself, ironically).